daemons/dss-sync-sfn/app.py

```diff
@@ -85,6 +86,37 @@ def launch_from_forwarded_event(event, context):
             raise NotImplementedError()
     return executions
 
+# This entry point is for operator initiated replication
+@app.sqs_queue_subscriber("dss-sync-operation-" + os.environ['DSS_DEPLOYMENT_STAGE'], batch_size=1)
```

Comment:
Consider increasing `batch_size` to amortize the startup cost in this SFN dispatcher. When triggered by an operator, the latency of aggregating a batch in the queue doesn't seem to be as much of an issue.

Reply:
`batch_size` changed to default.